src/gui/tray/usermodel.cpp

```diff
@@ -1364,6 +1371,28 @@ void UserModel::openCurrentAccountFolderFromTrayInfo(const QString &fullRemotePa
     _users[_currentUserId]->openFolderLocallyOrInBrowser(fullRemotePath);
 }
 
+void UserModel::openCurrentAccountFeaturedApp()
+{
+    if (!currentUser()) {
+        return;
+    }
+
+    if (!currentUser()->isFeaturedAppEnabled()) {
+        qCWarning(lcActivity) << "There is no feature app enabled on" << currentUser()->server();
+        return;
+    }
+
+    if (currentUser()->isNcAssistantEnabled()) {
+        QDesktopServices::openUrl(QUrl(_users[_currentUserId]->server(false).append("/apps/assistant/")));
```

Comment:
we may want to use `QUrl::fromUserInput` unless we really consider that `_users[_currentUserId]->server(false)` is fully trusted